.github/CONTRIBUTING.md

```diff
@@ -119,10 +119,10 @@ git checkout main
 git pull upstream main
 ```
 
-4. Install the dependencies with yarn (npm isn't supported):
+4. Install the dependencies with pnpm (npm/yarn isn't supported):

```

Comment:
can't we make it compatible with npm or yarn ? 
Should we make an announcement on contributors channel ?

Reply:
I can try and find a way to prevent use of yarn and npm